### ISSUE TITLE
flask-limiter breaks with werkzeug 0.10

### ISSUE THREAD

**Issue by g-p-g** [alisaifee/flask-limiter#25]
Hello,

werkzeug released a 0.10 version and now flask-limiter no longer works. This is because there is some logic for patching HTTP_STATUS_CODE when werkzeug is older than "0.9", the problem is that such dict doesn't exist at 0.10 and "0.10" <  "0.9".

**Comment by g-p-g** [comment]
```
import pkg_resources
d = pkg_resources.get_distribution("werkzeug")

vcheck = pkg_resources.parse_version('0.9')
vthis = d.parsed_version

print d.version
print d.version < '0.9'
print vthis < vcheck
```

**Comment by alisaifee** [comment]
Fixed with #24
